packages/widgets/src/commandpalette.ts

```diff
@@ -1128,74 +1137,91 @@ namespace Private {
     // Set up the result variables.
     let categoryIndices: number[] | null = null;
     let labelIndices: number[] | null = null;
+    let matchType = Infinity;
     let score = Infinity;
 
-    // Test for a full match in the category.
-    let cMatch = StringExt.matchSumOfDeltas(category, query);
-    if (cMatch && cMatch.score < score) {
-      score = cMatch.score;
-      categoryIndices = cMatch.indices;
-      labelIndices = null;
+    // First, test for an exact match in the label
+    if (LABEL_EXACT_MATCH_TYPE <= matchType) {
```

Comment:
Using the above enum type, this would become `if (matchType === MatchType.LabelExact) { ... }`